### DISCUSSION TITLE
Initial and objective images input

### DISCUSSION THREAD

**Discussion by babycommando** [pollinations/pollinations#82] [Pollinations]
Would be incredible to add a "seed" image and a "objective" image to the parameters as in this [Colab](https://colab.research.google.com/drive/1go6YwMFe5MX6XM9tv-cnQiSTU50N9EeT#scrollTo=mFo5vz0UYBrF)

Thanks.

**Comment by voodoohop** [comment]
@babycommando We are almost finished with milestone 0.1 (https://github.com/pollinations/pollinations/milestone/1) which guarantees minimal functionality. As soon as this is done we will add more advanced notebooks with more configurability and things such as seed images, etc..

**Reply by babycommando** [reply]
Awesome, thanks and congrats for the work! 

<pre>
                                                                                                    
                                                                                                    
                                                                                   -ydds.           
                                                        -so.                       yMMMN+           
                                                       :mMNs`                      -sys/`           
                                                       sMMMN-                                       
        `.`                   `-//-`                   :dNNh.                                       
      .ohhhy+.`              -hNMMms.                   -::.                                        
     .dhdmddmmy:`           `dMMMMMMd:             ````         ./o+:`                              
     yNsmNNmddmNh/`         `hNMMMMMMh`        `./syhhyo/.    `omNmmmd+.                            
    `mNhdNmNNNdhmNd+.`       .sdNNNNmo`      .+hmNmmmmmNNd+` .hNmmmmmNNh-        `:+ooo/-           
    `mMmhNmdNNNmyyNNms:`       .://:.      .+dNmmmmmmmmmNNN/`ymmmmmNNNNNh.`````:ymmmNddMNo`         
     sMNdmNmymNNNhomNNNms-               `:mNmmmmNNNNmmmNNN++mmmmmNNNNNNN/`  .yNMh:hNmomMN/         
     .mMmymNmydNNNhomNNNNNy:`            /NNmmNNNNNNNNNNNNm+ymNNNNNNNNNNd/./ssmMMd:hMmohMMy         
      yMNyhmmmddNNNyyNNNNNNmy-         `:mNmmNNNNNNNNNNNmNm/hmmNNNNNNNdo:omNNsdMMd:yMmoyMMh         
      :NNddNmddddNNmydNNNNNNNmo.  ```  `hNmmNNNNNNNNNNNNNNd/dNNNNNNNms/+dNNNNhyNMd:hMNohMMs         
      `yMdhNNNmdhdNNdsmNNNNNNNNd/````  :NNmmNNNNNNNNNNNNNNd+mNNNNNNd++hdmNNNNmsNNd:hMm+dMN-         
       :NmymNNNNmhhmmyyNNNNNNNNNm+`````oMNmNNNNNNNNNNNNNNNd+mNNNNNh+ymmhmNNNNNsmNd:dMdoNNo          
       `yNhmNNNNNmdyddohNNNNNNNNNmo````sNNNNNNNNNNNNNNNNNNhoNNNNmy+hNNNddNNNNNydNh/mNyhNh`          
        -mmhNNNNNNNmhdyodNNNNNNNNNmo`  +NNNNNNNNNNNNNNNNNNysNNNmy+hNNNNmdmNNNNhdNh+NNymd.           
         /NddNNNNNNNmmh+smNNNNNNNNNm+` -dNNNNNNNNNNNNNNNNNyyNNmy+hmmNNNNmdNNNNdhNysNmdd-            
          oNhdNNNNNNNNmy/yNNNNNNNNNNd/``sNNNNNNNNNmmNNNNNNshNNy+dmmmmmmNmmmNNNmhNyyNmm:             
          `sNhdNNNNmmmNms+dNNNNNNNNNNh- :dNNNNNmy+//+ymNNNsmNy+dNmmmmmmmmmmmNNmhNyhNm/              
           `smhdNNNNNNNNd/smNNNNNNNNNNs``omNNNd/``-:.`:hNmyNh+hNNmmmmmmmmmmmNNmhmsmNs`              
            `+mddNNNNNNNms/hNNNNNNNNNNd: :dNMN+ `oNNm/ -dNdm+ymNNmmmmmmmmmmmmNmddsNd.               
    `.--.`    :dmdmNmNNNNd:omNNNNNNNNNNo`.yNNd- -mMMMo  sNNhsmNNmmmmmmmmmmmmmNNmydm+                
 `+dNNNNmmdy+-`.smmmmmNNNm+/dNNNNNNNMNNh. /NNm/ `+hh+` `sNm++hmNNmmmmmmmmmmmmNNdsNy`     ```        
`hNmmmmmmmmNNmdo-:hmNNNNNNs-yMNo--/ymNMm/ .dNNy:` `  `-+dms``oydNNmmmmmmmmmmmNNhdd. -/oyhhddhyys+/- 
/NmmmmmmmmmmmNNNmo-+dNNNNNh-+NN/ .s/:yNNs``oNNNy.   `+dNNm+ :NNhhmmmmmmmmmmmmmmhm:`smmNNmmmmmmmmmmmy
/NmmmmmmmmmmmmmmNNd+:smNNNm/-mN+ :Mm/-hNy. -dNMm/`  -dNMNm: /NMmsdmmmmmmmmmmmmdmo.ymmmmmmmmmmmmmmmmN
-NhmmmmmmmmmmmmmNNNmy:+dNNNo.yNs`.mNo`+Ny. `sNMNy.  :mMNNy. +NMNoymmmmmmmmmmmmms-ymmmmmmmmmmmmmmmmmm
`hhhNmmmmmmmmmmmmNNNNd+/dNNd:omd:`-/-`/Nd-  :mMMm-  :mMMdo` oMMNssdmmmmmmmmmmmh-ymmmmmmmmmmmmmmmmmmN
 -dyhmNmmmmmmmmmmmNNNNms:yNNoohdyoooo+dNm/  `sMMN:  .mMMh+` oMMNo+dmmmmmmmmmmd/smmmmmmmmmmmmmmmmmmmm
  :dddNNmmmmmmmmmmmmmmNNy-yNhydssdNNNNNNN+   +MMN:  `dMMh/  sMMm:/dmmmmmmmmmm+ommmmmmmmmmmmmmmmmmmd:
   -hmdmmmmmmmmmmmmmmosmNh:+yhNd/+mNNNNNNo   :NMm-  `yMMd/ `hMMh`+mmmmmmmmmmsommmmmmmmmmmmmmmmmmmy- 
    `ommmmmmmmmmmmmmmms/omd/-oNNh:+mNNNNNs`  `++:`   /NNy. `dMN+`smmmmmmmmms+dmmmmmmmmmmmmmmmmmd+`  
      -smmddmmmmmmmmmmmdo:oho-oNNy-/dNNNNy`  ``..`   `/+-  `dNh.-dmmmmmmmms/dmmmmmmmmmmmmmmmmdo.    
        `/hdyhdmmmmmmmmNNmy/os:oNms.-yNNNy` `sdmmy.   --`   -:. +mmmmmmmms:dmmmmmmmmmmmyyhdh+.      
           -smhyshdddmNNmmNmh++//dmh:.sNNy``+NMMMNh-`:hmh/`    .ymmmmmmmo:dmmmmmmmmmdsodd:.`        
             .odNdysyyyhmNNNmmhs/-oddo.+mh./mMMMMMNdhNMMMN/   `smmNmmmd+/dmmmmmmmmy+odms.           
               `-odNmhysssydmmmmmdo/+hy/+d/yNMMMdyhmMMMMMMs  `omNNNmmh/odmmmmmdyo+ymh+.             
                  `-ohmNmdhsosydmmmmhooo/oyhNMMh-`.+NMMMMN+  -mNNNNdo/ymmmdyo+sdmds-`               
                     ``:oydmNmdhyyyhmmmy+.-omNMmy/+yNMMMNs`  `+yhho-:hdhssshddy+-`                  
                          `.:/osyyyyhhhs/` `:smNMNNMMNmy:`  `..--.  :yhhhhs+-.`                     
                                   ````..``` `-/+oo+/:.    /ydmmd+` `.-..`                          
                                     .ymmdhhy:          `  :dMMMMy`                                 
                                     oMMMMMMMd.      `:oyo-`:hNNm/                                  
                                     :dNMMMNd+`      /mMMNm/ ./+:`                                  
                                      .:++/:`   -:`  -ymNmh/                                        
                                               `yms`  `...`                                         
                                               .mMm.                                                
                                               `dMm-                                                
</pre>
